### PR TITLE
refactor: rename swagger to oas in errors

### LINKED ISSUE #164: Rename error messages from swagger to openapi

### PR DESCRIPTION
**Pull request by fredmaggiowski**
resolves #164

### CODE REVIEW

**Review by davidebianchi — COMMENTED** [review]
Nice! There are also some error messages in the `generateNewValidOpenapi` function

**Comment by fredmaggiowski** [comment]
> `generateNewValidOpenapi`

I should have changed all occurrences of swagger to openapi 😓

File: main.go [hunk 1]
```diff
@@ -13,21 +13,21 @@ import (
...
 
 var (
-	// ErrGenerateSwagger throws when fails the marshalling of the swagger struct.
-	ErrGenerateSwagger = errors.New("fail to generate swagger")
-	// ErrValidatingSwagger throws when given swagger params are not correct.
-	ErrValidatingSwagger = errors.New("fails to validate swagger")
+	// ErrGenerateOAS throws when fails the marshalling of the swagger struct.
+	ErrGenerateOAS = errors.New("fail to generate openapi")
+	// ErrValidatingOAS throws when given openapi params are not correct.
+	ErrValidatingOAS = errors.New("fails to validate openapi")
```
Comment (davidebianchi) [inline main.go:19]: Those values are exported. Maybe I'd continue to export also the old errors with a deprecation warning. What do you think about?

Comment (fredmaggiowski) [reply main.go:19]: makes sense

File: main.go [hunk 2]
```diff
@@ -13,21 +13,26 @@ import (
...
+
+	// Deprecated: ErrGenerateSwagger has been deprecated, use ErrGenerateOAS instead
```
Comment (fredmaggiowski) [inline main.go:21]:
```suggestion
	// Deprecated: ErrGenerateSwagger has been deprecated, use ErrGenerateOAS instead.
```

**Review by davidebianchi — APPROVED** [review]
LGTM

**Pull request merged**